sympy/polys/polytools.py

```diff
@@ -6744,17 +6827,23 @@ def all_roots(f, multiple=True, radicals=True):
     >>> [r.evalf(3) for r in all_roots(p)]
     [1.17, -0.765 - 0.352*I, -0.765 + 0.352*I, 0.181 - 1.08*I, 0.181 + 1.08*I]
 
-    Irrational algebraic or transcendental coefficients cannot currently be
-    handled by :func:`all_roots` (or :func:`~.rootof` more generally):
+    Irrational algebraic coefficients are handled by :func:`all_roots`
+    if `extension=True` is set.
 
     >>> from sympy import sqrt, expand
     >>> p = expand((x - sqrt(2))*(x - sqrt(3)))
     >>> print(p)
     x**2 - sqrt(3)*x - sqrt(2)*x + sqrt(6)
-    >>> all_roots(p)
-    Traceback (most recent call last):
-    ...
-    NotImplementedError: sorted roots not supported over EX
+    >>> all_roots(p, extension=True)
+    [sqrt(2), sqrt(3)]
```

Comment:
I think it is good to show both examples here i.e. what happens with and without `extension=True`.

Reply:
Done